pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -80,13 +83,15 @@ func getRunName(pr *v1beta1.PipelineRun) string {
 func getPipelineRunController(t *testing.T, d test.Data) (test.Assets, func()) {
 	//unregisterMetrics()
 	ctx, _ := ttesting.SetupFakeContext(t)
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
ohhh :sweat:

Reply:
Not sure I understand...

Reply:
@mattmoor it was me realizing where I did put that `context.WithCancel` before :wink: